### PR TITLE
Apply Button on Benefit info page

### PR DESCRIPTION
**Pull request by nghe0001**
# Description 📝

Apply Button on Benefit info page

## Taiga Link 🔗

https://taiga.dts-stn.com/project/ericdube-hfp-shared-backlog/task/263?kanban-status=131

## Checklist ✅
- [ ] created story for all visual components
- [ ] created or modified necessary unit tests
- [ ] created or modified e2e tests
- [ ] modified README or any applicable documentation on the changes you made

### CODE REVIEW

**Review by will0684 — APPROVED** [review]
Thanks Kevin

File: src/components/organisms/BenefitGrid.stories.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ Primary.args = {
       isEligible: true,
     },
   ],
-  benefitMoreInfoButtonText: "More Information",
+  benefitMoreInfoButtonText: "More Informationaaa",
```
Comment (will0684) [inline src/components/organisms/BenefitGrid.stories.js]: This looks like a typo

**Review by will0684 — CHANGES_REQUESTED** [review]
Once the typo is fixed should be good to merge

File: src/components/organisms/BenefitGrid.stories.js (hunk 1 above)
Comment (nghe0001) [reply src/components/organisms/BenefitGrid.stories.js]: corrected ty

**Review by will0684 — APPROVED** [review]
Nice thanks Kevin

**Pull request closed without merging**